packages/cashier/src/components/cashier-container/real/real.scss

```diff
@@ -0,0 +1,23 @@
+.real {
```

Comment:
I think maybe we can add a new prop to the breadcrumbs named `className` and then use a class for having these styles as we have duplicated these styles for the breadcrumbs in two different places.

P.S: If it's not in the scope of this card just add `TODO` for these two duplicate styles and ignore the comment.

Reply:
@hamid-deriv definitely agree with you, will add TODO for now and create an improvement card for Breadcrumb component in deriv-ui